quantum/keymap_extras/keymap_hebrew.h

```diff
@@ -0,0 +1,151 @@
+/* Copyright 2020
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "keymap.h"
+
+// clang-format off
+
+/*
+ * ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┐
+ * │ ; │ 1 │ 2 │ 3 │ 4 │ 5 │ 6 │ 7 │ 8 │ 9 │ 0 │ - │ = │ \ │   │
```

Comment:
Aaand I just noticed these box drawings are inconsistent. Do Hebrew keyboards still use BAE or is it usually the standard ISO enter these days?

Reply:
That's a good question.

Reply:
Yup, seems like the diagram at Wikipedia is out of date vs. the real world. 
I have updated to ISO Enter, and switched IL_BSLS to use KC_NUHS, which is similar to Greek, Latvian and Lithuanian QWERTY which have backslash + pipe on that key. I've tested on Mac and ChromeOS.